cscore/src/main/java/edu/wpi/first/cscore/CameraServerCvJNI.java

```diff
@@ -66,6 +66,8 @@ public static native int createCvSource(
 
   public static native int createCvSink(String name);
 
+  public static native int createCvSinkWPixelFormat(String name, int pixelFormat);
```

Comment:
I think it's fine to modify createCvSink()?  We don't in general promise stability of the low-level JNI functions.

Reply:
The reason I left a fallback version of createCvSink is because that way I can leave default behavior up to CvSinkImpl.cpp rather than having a default assigned in both places

Reply:
Unless we want the Java library to be forced to select a pixelFormat

Reply:
I'm okay with putting the default in both places--we do that in a number of places, and it's easier to maintain one JNI entry point than two.